solc.ts

```diff
@@ -51,46 +53,45 @@ program
 program.parse(process.argv);
 const options = program.opts();
 
-var files = program.args;
-var destination = options.outputDir || '.'
+const files = program.args;
+const destination = options.outputDir || '.';
```

Comment:
I see a lot of simple change and reformatting that would still work even in the JS version. Could you extract them into a separate PR? They should pass through review pretty easily and here we could focus on the actual TypeScript-related changes.

Reply:
I have gone and rolled back all the linting changes. Drastically reduced the changes made.